components/BottomAppBar/docs/accessibility.md

```diff
@@ -0,0 +1,62 @@
+To help ensure your bottom app bar is accessible to as many users as possible, please be sure to review the
+following recommendations:
+
+### Set `-accessibilityLabel`
+
+Set an appropriate
+[`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel)
+to all buttons within the bottom app bar.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+bottomAppBar.floatingButton.accessibilityLabel = "Compose"
+let trailingButton = UIBarButtonItem()
+trailingButton.accessibilityLabel = "New"
+bottomAppBar.trailingBarButtonItems = [ trailingButton ]
+```
+
+#### Objective-C
+
+```objc
+bottomAppBar.floatingButton.accessibilityLabel = @"Compose";
+UIBarButtonItem *trailingButton = 
+    [[UIBarButtonItem alloc] initWithTitle:nil
+                                     style:UIBarButtonItemStylePlain
+                                    target:self
+                                    action:@selector(didTapTrailing:)];
+trailingButton.accessibilityLabel = @"Buy";
```

Comment:
This may be the place to unify the strings.

Reply:
Done